### ISSUE TITLE
Support JSON input in cromwell command

### ISSUE THREAD

**Issue by prihoda** [chanzuckerberg/miniwdl#165]
It would be useful to accept also the traditional `-i, --inputs` JSON input in the `cromwell` command. In some cases, the user might have the JSON files ready and it would be a pain to rewrite them as arguments. This JSON file would serve as a base dictionary that would be extended (and potentially overwritten) with the values supplied in the args.

Multiple JSON input files could actually be supported, this would be useful for supplying platform-specific defaults:
```bash
miniwdl cromwell hello.wdl -i hello.local.json -i hello.world.json greeting=Hi
```

**Comment by prihoda** [comment]
This also applies to the cromwell `--options` argument.

**Comment by mlin** [comment]
@prihoda I've implemented this in v0.2.1. It wasn't immediately convenient to support multiple sequentially merged JSON files just due to some vagaries of how we're representing the namespaced inputs, but it can be done eventually. Thanks!
